jbpm-recommendation-smile-naivebayes/src/test/java/org/jbpm/prediction/naivebayes/Series1Test.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.prediction.naivebayes;
+
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class Series1Test extends NaiveBayesRecommendationServiceProcessTest {
+    /**
+     * shows how after passing min count of 2 input, accuracy goes to > 0.95 (95%) very quickly.
+     */
+    @Test
+    public void test1() {
```

Comment:
Test method name is meaningless: instead of adding the comment, name the method with the test behavior.
E.g. (or something similar):
when_AtLeastTwoInputWithSameLevel_Expect_HighConfidenceForApproval